### DISCUSSION TITLE
raw.githubusercontent.com adding filename with directory for non-human-readable files

### DISCUSSION THREAD

**Discussion by dhalbert** [community/community#17023] [Other Feature Feedback, Questions, & Ideas]
We're not sure if this behavior is old or new. Do you know?

This kind of URL:
https://raw.githubusercontent.com/adafruit/Adafruit_CircuitPython_CircuitPlayground/main/examples/dip.wav
is causing a browser to download the file as `examples_dip.wav`.

The reason is that the HTTP headers are specifying `examples/dip.wav` as the `filename`:
```sh
$ curl -I https://raw.githubusercontent.com/adafruit/Adafruit_CircuitPython_CircuitPlayground/main/examples/dip.wav
HTTP/2 200 
...
content-disposition: attachment; filename=examples/dip.wav
...
```
Since the browser (Chrome or Firefox) won't create the `examples` directory, it changes the `/` to `_`, and downloads `examples_dip.wav`.

We want the user to be able to download `dip.wav` directly, with its original name, and have webpages with links like that. (The original URL is actually `https://github.com/adafruit/Adafruit_CircuitPython_CircuitPlayground/raw/main/examples/dip.wav`, but it redirects to `raw.githubusercontent.com`.)

If you happen to know this behavior is new, I'll report it as a bug. Otherwise maybe it never worked the way we thought.

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
